docs/networking.md

```diff
@@ -42,6 +42,7 @@ Several different CNI providers are currently built into kops:
 * [romana](https://github.com/romana/romana)
 * [weave](https://github.com/weaveworks/weave-kube)
 * [amazon-vpc-routed-eni](./networking.md#amazon-vpc-backend)
+* [Cilium](http://docs.cilium.io/en/v1.1/)
```

Comment:
@nebril you might want to just point to docs.cilium.io and let the URL redirect to the latest stable release.